File: Words-Android/app/src/main/java/com/example/al333z/words/service/Word.java
package com.example.al333z.words.service;

public class Word {

    public final long id;
    public final String word;
    public final int day;
    public final int month;
    public final int year;
    public final String imageUrl;

    public Word(int id, String word, int day, int month, int year, String imageUrl) {
        this.id = id;
        this.word = word;
        this.day = day;
        this.month = month;
        this.year = year;
        this.imageUrl = imageUrl;
    }
}

File: Words-iOS/Words/Service/Word.swift
import Foundation

struct Word {

    let id: Int
    let word: String
    let day: Int
    let month: Int
    let year: Int
    let imageUrl: String

    init(id: Int, word: String, day: Int, month: Int, year: Int, imageUrl: String){
        self.id = id
        self.word = word
        self.day = day
        self.month = month
        self.year = year
        self.imageUrl = imageUrl
    }
}

extension Word: Printable {
    var description: String {
        return "Word \(id) - \(word) - \(day)/\(month)/\(year) - \(imageUrl)"
    }
}